doc/src/manual/strings.md

```diff
@@ -516,9 +514,8 @@ false
 julia> contains("Xylophon", 'o')
 ERROR: MethodError: no method matching contains(::String, ::Char)
 Closest candidates are:
-  contains(!Matched::Function, ::Any, !Matched::Any) at reduce.jl:640
-  contains(::AbstractString, !Matched::AbstractString) at strings/search.jl:366
- ...
+  contains(::Function, ::Any, ::Any) at reduce.jl:628
```

Comment:
Right, I mean the child-process worker version of Julia that executes the jldoctest-fenced commands, not necessarily the master doctest process that shows you whether or not they pass.